config/running.go

```diff
@@ -26,19 +26,19 @@ type (
 	}
 )
 
-// loadRunningConfig attempts deserializes data in the static config
-func loadRunningConfig(config *StaticCfg) (*RunningCfg, error) {
-	var outConfig = new(RunningCfg)
+// initRunningConfig uses data in the static config initialize
+// the passed in running config
+func initRunningConfig(static *StaticCfg, running *RunningCfg) error {
```

Comment:
This change wasn't strictly necessary and was just a refactor to match the usage of the `parseStaticConfig` function.  I can revert it if needed.